spec/features/step_by_step_actions_spec.rb

```diff
@@ -76,8 +75,19 @@ def then_there_should_be_no_primary_action
   end
 
   def and_the_secondary_action_should_be(action_text)
-    custom_error = "Couldn't find '#{action_text}' as a secondary action in: \n #{action_html}"
-    expect(page).to have_css(".app-side__actions .gem-c-button--secondary", text: action_text), custom_error
+    expect(page).to have_css(secondary_action_selector, count: 1)
+    assert_secondary_action_exists(action_text)
+  end
```

Comment:
It feels like there is duplication between the `and_the_secondary_action_should_be` and `and_the_secondary_actions_should_be` methods since they both test the same thing (just one takes a string, the other an array).  Maybe there could be one method which accepts an array like you've done for `and_there_should_be_tertiary_actions_to`, then have single item arrays when this method is invoked?  That would also mean `assert_secondary_action_exists` wouldn't be needed.

Reply:
Have now made the approach consistent with how we do tertiary actions.